### ISSUE TITLE
Start the year ticks at 1800

### ISSUE THREAD

**Issue by bkleeman** [rcc-uchicago/sankey#6]
The year ticks on the slider currently begin at 1805. They should begin at 1800.
![image](https://user-images.githubusercontent.com/20052075/81234888-5113d080-8fbf-11ea-8cba-0023f5379151.png)

**Comment by Milstein** [comment]
![image](https://user-images.githubusercontent.com/144645/81401622-53784680-90f5-11ea-9760-e79f47beb149.png)

I don't know what alignment of year and slider year pointer going to look like?

**Comment by bkleeman** [comment]
Ah I think I see what you mean. Like, the year on #dynamicYear/output/“moving date” might not appear directly above the 1800 tick mark when the year is set to 1800 in the slider — is that what you’re saying?

Also, I tried adjusting the first value of that array of years, the one that is used to set the tick marks above the slider. I tried changing it to 1800 and it the number was cut off on the left. I noticed that the years array is funneled through several mathematically complex functions that are over my head, at least for now. 

Does anyone have a solid understanding of how that process works?

**Comment by Milstein** [comment]
Yes, the array stores the years axis on top  to display- you had already did the testing what I am referring that is not possible as the top axis should not be showing cut of 1800 value.
